### PR TITLE
Small API documentation fixes

### PR DESCRIPTION
**Pull request by andy-armstrong**
## Description

I've made a few more tweaks to the UI Toolkit based upon changes to the Pattern Library:
 - upgraded to the new svg logo
 - fixed up the spacing on the footer
 - changed code examples to wrap
 - added a 404 page

You can see the new doc site here:

http://ux-test.edx.org/andya/fix-edx-logo

A 404 on S3 doesn't redirect to the custom page, but you can view it here:

http://ux-test.edx.org/andya/fix-edx-logo/404.html

## Post-review
- [ ] Squash commits into discrete sets of changes with descriptive commit messages.

## Reviewers
- [x] @AlasdairSwan 
- [x] @dsjen 

FYI @bjacobel 

If you've been tagged for review, please check your corresponding box once you've given the :+1:.

We'll also automatically suggest reviewers for this PR based on the code it touches.

### CODE REVIEW

**Comment by AlasdairSwan** [comment]
Nitpick: At the smallest breakpoints there is still no padding for the footer.

**Comment by AlasdairSwan** [comment]
+1 for the SVG icon. Once padding has been added for the footer at all breakpoints 👍

File: doc/_layouts/default.html [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
             <div class="doc-wrapper-content-main">
                 <main class="doc-content-main" id="content-main" aria-label="Content">
-                    {% if page.viewClass != 'index' %}
+                    {% if page.githubPath %}
```
Comment (dsjen) [inline doc/_layouts/default.html]: Nice! 💯

**Comment by dsjen** [comment]
LGTM! 👍   I have the same thought on @AlasdairSwan on the footer.

**Comment by andy-armstrong** [comment]
Thanks @AlasdairSwan and @dsjen. I've fixed the footer padding issue and pushed up a preview:

http://ux-test.edx.org/andya/fix-edx-logo/

I'll go ahead and merge once I get a green build.

**Pull request merged**